android/src/main/java/com/magicleap/magicscript/scene/nodes/video/VideoNode.kt

```diff
@@ -143,6 +194,15 @@ class VideoNode(
         nodeClipper.applyClipBounds(this, clipBounds)
     }
 
+    private fun createSubtitles() {
+        if (subtitles == null) {
+            subtitles = UiSubtitlesNode(
+                    props = getSubtitleProps(), context = context, viewRenderableLoader = viewRenderableLoader, nodeClipper = nodeClipper, fontProvider = fontProvider)
```

Comment:
This line is probably too long, lets put each property in a new line. Also the closing bracket should be in the new line ;-)